### ISSUE TITLE
Update to work without mixins

### ISSUE THREAD

**Issue by rwwagner90** [cbroeren/ember-context-menu#56]
@cbroeren would you be open to updating this to not use mixins and instead set the `contextmenu` call up directly? I have it working in my app with `{{on "contextmenu" this.contextMenuTrigger}}`

**Comment by cbroeren** [comment]
@rwwagner90 yes, I'm totally open do to so. Let me see if I get some time to do so later this week.

**Comment by rwwagner90** [comment]
@cbroeren sounds good! I'm happy to help if you'd like. I have it working in my app.
